source/extensions/filters/http/jwt_authn/extractor.cc

```diff
@@ -329,11 +328,6 @@ absl::string_view ExtractorImpl::extractJWT(absl::string_view value_str,
     return value_str;
   }
 
-  // There should be two dots (periods; 0x2e) inside the string, but we don't verify that here
```

Comment:
I think you are reverting the previous PR, which is not the intention.
Runtime-guards are used as temporary flags that one puts around a user-facing change. Consider a scenario where someone upgrades an Envoy, and observes that the change is breaking their use-cases. They can temporarily revert the flag's value, and come up with a fix to update their deployment.

After 6 months, the runtime-flag (and old-code) is deprecated from Envoy.

Reply:
I removed relevant parts of my runtime guard.
in this area removed my condition that check if my runtime flag is enabled.
About the comment it's a mistake and I returned it.